bin/DMHS_ATS_configurable

```diff
@@ -45,13 +39,17 @@ def cmdline():
                         help="Port number for webserver")
     parser.add_argument("--send_efd_message", action="store_true", default=False,
                         help="Send and EDF message as well?")
+    parser.add_argument("--log", action="store", default='INFO',
```

Comment:
I think I would prefer `--loglevel` or `--log-level` as the name of this parameter. That's what the parameter becomes internally.

Reply:
`--loglevel` was my original parameter, but the changed it to `--log `because it looked simpler. Happy to change it `--loglevel`

Reply:
loglevel is consistent with what science pipelines are doing (pytest uses `log-level` so there isn't consistency for python).